pom.xml

```diff
@@ -227,7 +227,25 @@
                 <goal>sort</goal>
               </goals>
             </execution>
-          </executions>
+          </executions>                                             
+        </plugin>
```

Comment:
Have you executed the new XML formatter plugin on the code? I wonder why in some files it is using spaces and in others tabs?

Reply:
i had just edited these manually